### PR TITLE
feat: add github action to deploy es-ds-v2

### PR DESCRIPTION
**Pull request by RANE1021**
### 🔗 Linked issue
- [CED-1391](https://energysage.atlassian.net/browse/CED-1391): Parallel deploy setup for version 2

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [x] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description
-  Added GitHub action to automate build process
-  changed necessary info to point to to new url

### 🥼 Testing
- GitHub actions passes
- files are uploaded to S3 and changes are reflected on website

#### 🧐 Feedback Requested / Focus Areas

-

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.
- [ ] I have documented testing approach

[CED-1391]: https://energysage.atlassian.net/browse/CED-1391?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by ericdouglaspratt — CHANGES_REQUESTED** [review]
Just to capture for others what we discussed - first steps here before review are:
- Get CI to pass
- Move the `es-cdk` include from the root package.json to the aws/package.json

File: es-design-system/aws/package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   "dependencies": {
     "aws-cdk": "^2.69.0",
     "aws-cdk-lib": "^2.69.0",
-    "es-cdk": "^1.20.0",
```
Comment (tomleo) [inline es-design-system/aws/package.json:14]: I'd keep this on the same version as `main` for now if possible.

File: es-design-system/package-lock.json [hunk 2]
```diff
@@ -2682,18 +2679,18 @@
             }
         },
         "node_modules/@energysage/es-bs-base": {
-            "version": "2.0.0-alpha.0",
-            "resolved": "https://registry.npmjs.org/@energysage/es-bs-base/-/es-bs-base-2.0.0-alpha.0.tgz",
-            "integrity": "sha512-aijqFOPrPgf3iNp7ccqJHDGDgDkiyHduyQV7fxheQU5A+73rSMQ4JXG/z81pXAX7qTabI1A90rtmyJd8GXEIOw==",
+            "version": "2.0.0-alpha.1",
+            "resolved": "https://energysage-659694780082.d.codeartifact.us-east-1.amazonaws.com/npm/cloud-engineering/@energysage/es-bs-base/-/es-bs-base-2.0.0-alpha.1.tgz",
```
Comment (tomleo) [inline es-design-system/package-lock.json:2683]: It looks like some of these dependencies are tied to codeartifact still

File: es-design-system/aws/package.json (hunk 1 above)
Comment (RANE1021) [reply es-design-system/aws/package.json:14]: It is.

File: es-design-system/package-lock.json (hunk 2 above)
Comment (RANE1021) [reply es-design-system/package-lock.json:2683]: Not sure why it keeps doing that

**Comment by RANE1021** [comment]
@ericdouglaspratt all set now

**Review by ericdouglaspratt — APPROVED** [review]
Tested successfully!

**Pull request merged**